src/gbinder_remote_reply.c

```diff
@@ -73,15 +73,12 @@ gbinder_remote_reply_set_data(
     GBinderRemoteReply* self,
     GBinderBuffer* buffer)
 {
-    if (G_LIKELY(self)) {
-        GBinderReaderData* data = &self->data;
+    /* The caller checks the pointer for NULL */
```

Comment:
There is gbinder_remote_reply_set_data(NULL, NULL) in test code - maybe it should be removed?

Reply:
It is removed